distribution/docs/src/main/resources/_contents/_plugins/metacardingest-network-contents.adoc

```diff
@@ -0,0 +1,75 @@
+==== Metacard Ingest Network Plugin
+
+Allows the conditional insertion of new attributes on metacards during ingest based on network information from the ingest request; including IP address and hostname.
+
+For the extent of this section, a 'rule' will refer to a configured, single instance of this plugin.
+
+===== Installing the Metacard Ingest Network Plugin
+
+The Metacard Ingest Network Plugin is installed by default during a standard installation in the catalog application.
+
+===== Configuring the Metacard Ingest Network Plugin
+
+Multiple instances of the plugin can be configured by clicking on its configuration title within the configuration tab of the catalog app. Each instance represents a conditional statement, or a 'rule', that gets evaluated for each ingest request. For any request that meets the configured criteria of a rule, that rule will attempt to transform its list of key-value pairs to become new attributes on all metacards in that request.
+
+The condition piece of the rule is divided into two fields: "Criteria" and "Expected Value". The "Criteria" field features a drop-down list containing the four elements for which equality can be tested:
+
+* IP Address
+* Host Name
+* Scheme
```

Comment:
Should we have a description and/or example for those, especially for _Scheme_  and _Context Path_? Or maybe update the example to include those as well?